wrappers/msbuild

```diff
@@ -66,5 +66,10 @@ arm64)
     ;;
 esac
 
+if [[ "$(uname -m)" == "x86_64" ]]; then
```

Comment:
Is it better to test `"$BINDIR" contains "Hostx64"`?

Reply:
I've added a test for `$MSBUILDBINDIR` because it's easier to match.

Reply:
LGTM!